pkgs/tools/audio/mpdris2/default.nix

```
@@ -17,8 +20,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoreconfHook ];
   buildInputs = [ intltool pythonPackages.wrapPython ];
```

```suggestion
  buildInputs = [ intltool  ];
```

not needed with buildPythonApplication

---

intltool looks like it neds to be native also, as it's just used for configure.